### ISSUE TITLE
Stylelint raises function-calc-no-invalid when stylelint-config-drupal is used

### ISSUE THREAD

**Issue by openjck** [u7aro/stylelint-config-drupal#6]
### Steps to reproduce

1. Clone [this repo](https://github.com/openjck/bug-demo-stylelint-config-drupal-6)
2. Run `npm install`
3. Run `npm run lint:css`

### Actual result
```
example.css
1:1  ✖  Unknown rule function-calc-no-invalid  function-calc-no-invalid
```
### Expected result

No error

**Comment by DigitalFeonix** [comment]
For now I have updated my `package.json` to utilize the [overrides feature](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) to get the latest configs this package extends.

```json
{
  "devDependencies": {
    "stylelint": "^14.6.0",
    "stylelint-config-drupal": "^2.0.0"
  },
  "overrides": {
    "stylelint-config-standard": "^25.0.0"
  }
}
```

**Comment by u7aro** [comment]
@openjck @DigitalFeonix 

This issue is due to the major version of Stylelint being 14.

The Drupal core package.json has been updated, so I've merged the latest configurations to resolve this issue.

I released v3.0.0, so please try the following commands.

```shell
npm install stylelint-config-drupal@^3.0.0
```

**Comment by openjck** [comment]
That solved the problem for me. Thank you very much!
